### PR TITLE
chore: overhaul profile page - update old links, make more rich

### PR DESCRIPTION
**Pull request by olliegilbey**
image

Hey team, this PR alters the structure to make wayfinding the stack better and more rich for those arriving on the org.

Changed to the new Discord link.

See the above screenshot for before and after.

Props to @pyramation for all the iterations we spoke about. 🤍

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
 # Cosmos
 
-The Cosmos org is collection of teams working on decentralized technology.
+A collection of teams working on decentralized technology for the internet of blockchains - The Interchain.
 
-## Quick Start
+| [Developer Portal](https://tutorials.cosmos.network) - Quick Start | -  | [Interchain Discord](https://discord.com/invite/interchain) - For Support and Community |
```
Comment (julienrbrt) [inline profile/README.md]: Don't we want people to ask support question on GH discussion instead?
Who is actually looking at discord forum except Reece 😅

Comment (olliegilbey) [reply profile/README.md]: It's a good point 😋
I think for now at least, Discord is still the place with the highest support question volume, even with effectively just @Reecepbcups in the trenches - and we have some plans for helping Reece out.
We can also have further conversations on this, but for now, I still think Discord is the place to pass general support to, particularly for newer devs.

Comment (julienrbrt) [reply profile/README.md]: I'd argue that it is easier for us to check (I've muted discord personally) and at least the answers are indexed. So it avoids answering 15 times the same thing and gives better SEO.

Comment (reecepbcups) [reply profile/README.md]: I've been manually indexing common questions already from discord. So we can put these into guides or deliver on github for better SEO. There are still a lot more over the past year I have missed, but it's a good start.

Everyone likes discord (sadly). I think we should at least link to both the discord & GH forum

image

File: profile/README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
 # Cosmos
 
-The Cosmos org is collection of teams working on decentralized technology.
+A collection of teams working on decentralized technology for the internet of blockchains - The Interchain.
 
-## Quick Start
+| [Developer Portal](https://tutorials.cosmos.network): Quick Start | [Interchain Discord](https://discord.com/invite/interchain): Support & Community | [GitHub Discussions](https://github.com/orgs/cosmos/discussions): Technical Hub |
```
Comment (olliegilbey) [inline profile/README.md:5]: Adding GH Discussions to the top line here as the "Technical Hub" - should resolve the prior convo, what do we think team?

Comment (reecepbcups) [reply profile/README.md:5]: LGTM!

**Review by reecepbcups — APPROVED** [review]

File: profile/README.md (hunk 2 above)
Comment (julienrbrt) [reply profile/README.md:5]: 👍🏾

**Review by julienrbrt — APPROVED** [review]

**Pull request merged**